packages/viewer/src/components/viewer-spinner/viewer-spinner.css

```diff
@@ -0,0 +1,94 @@
+
+
+.spinner {
+  position: relative;
+  display: flex;
+  height: 16px;
+  width: 16px;
+}
+
+.spinner div {
+  position: absolute;
+  display: block;
+  animation: spinner 1.2s cubic-bezier(0.5, 0, 0.5, 1) infinite;
+  border-radius: 50%;
+  border-style: solid;
+  box-sizing: border-box;
+  border-color: var(--neutral-800) transparent transparent transparent;
+  border-width: var(--border-width, 1px);
```

Comment:
This CSS prop is not documented as part of the component. Also, please use the CSS prop naming conventions like the other components.